sopel/modules/ip.py

```diff
@@ -4,12 +4,13 @@
 # Copyright © 2013, Elad Alfassa <elad@fedoraproject.org>
 # Licensed under the Eiffel Forum License 2.
 
-from __future__ import unicode_literals, absolute_import, print_function, division
```

Comment:
Don't remove future imports. This line is part of Sopel's [style guide](https://github.com/sopel-irc/sopel/blob/master/CONTRIBUTING.md#committing-code).